### PR TITLE
Add remote repository support (Fixes #9)

### LINKED ISSUE #9: Support for a different target
It would be useful to be able to use this action not just for the current check run/suite, but also to create check runs on another repository and commit.

### PR DESCRIPTION
**Pull request by Saviq**

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -17,18 +17,33 @@ async function run(): Promise<void> {
     const octokit = github.getOctokit(inputs.token);
 
     const ownership = {
-      owner: github.context.repo.owner,
-      repo: github.context.repo.repo,
+      owner: '',
```
Comment (LouisBrunner) [inline src/main.ts:20]: Could you keep the default values the same? It makes it more explicit what will be the default and how/when it is overridden.

File: src/main.ts [hunk 2]
```diff
@@ -17,18 +17,33 @@ async function run(): Promise<void> {
...
+      repo: '',
     };
-    const sha = github.context.sha;
+    let sha = '';
+
+    if (inputs.repo) {
+      const repo = inputs.repo.split('/');
```
Comment (LouisBrunner) [inline src/main.ts:26]: Would be nice to be sure we can actually split it, would you mind adding a check in `src/inputs.ts`?

File: action.yml [hunk 3]
```diff
@@ -5,6 +5,11 @@ branding:
   icon: 'check-circle'
   color: 'green'
 inputs:
+  repo:
+    description: 'the target `owner/repo` to manage the check run on (defaults to the current repository'
+    required: false
+  sha:
+    description: 'the target commit''s SHA (defaults to the current commit)'
```
Comment (LouisBrunner) [inline action.yml:12]: Does it need to have `required: false` explicitly like the other?

**Review by LouisBrunner — CHANGES_REQUESTED** [review]
Thanks a lot for your contribution! I added a few comments around areas which could be improved a little, would you mind making those changes?

File: action.yml (hunk 3 above)
Comment (Saviq) [reply action.yml:12]: It doesn't ([false is default](https://docs.github.com/en/free-pro-team@latest/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_idrequired)), but I've added it for consistency - missed it before.

File: src/main.ts (hunk 1 above)
Comment (Saviq) [reply src/main.ts]: Right, not sure I complicated it like this. Fixed :)

File: src/main.ts (hunk 2 above)
Comment (Saviq) [reply src/main.ts:26]: And fixed, also added a test for this.

**Review by LouisBrunner — APPROVED** [review]

**Comment by LouisBrunner** [comment]
Thanks a lot :)

**Pull request merged**